exercises/tree-building/src/test/java/BuildTreeTest.java

```diff
@@ -0,0 +1,167 @@
+import org.junit.Test;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+public class BuildTreeTest {
+    @Test
+    public void test_emptyList_shouldReturnNull() throws InvalidRecordsException {
+        ArrayList<Record> records = new ArrayList<>();
+
+        TreeNode root = new BuildTree().buildTree(records);
+        assertNull(root);
+    }
+
+    @Test
```

Comment:
All tests except the first one should be ignored, as per our [policies](https://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-tests) 🙂